org.eclipse.xtext.core.idea.tests/.project

```diff
@@ -11,17 +11,12 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
```

Comment:
https://github.com/xtext/xtext-gradle-plugin/blob/9a0a1dafc415aacc62c763cee9c8bf82f9e285f9/xtext-gradle-plugin/src/main/java/org/xtext/gradle/XtextBuilderPlugin.xtend should add that